test/functional/apps/dashboard/_dashboard_clone.js

```diff
@@ -61,8 +61,7 @@ export default function ({ getService, getPageObjects }) {
 
     it('and warns on duplicate name', async function () {
       await PageObjects.dashboard.confirmClone();
-      const isWarningDisplayed = await PageObjects.dashboard.isDuplicateTitleWarningDisplayed();
-      expect(isWarningDisplayed).to.equal(true);
+      await PageObjects.dashboard.expectDuplicateTitleWarningDisplayed({ displayed: true });
```

Comment:
I don't know if dashboard.confirmClone() is waiting until the action is complete and in that case dashboard.expectDuplicateTitleWarningDisplayed() should instantly find the duplicate warning.
Or if we're expecting dashboard.expectDuplicateTitleWarningDisplayed() to wait for it, and for how long?

Reply:
It'll wait up to 60 seconds to find the warning, so I think that is an appropriate indicator that the action finally completed.